development-testing/antiskid.py

```diff
@@ -0,0 +1,14 @@
+import os, sys, ctypes
+#This script will detect if the stager is run in a vm or not.
+#Windows only! This is basicly a pre stager.
+stream = os.popen('WMIC COMPUTERSYSTEM GET MANUFACTURER')
+output = stream.read()
+#string cleaning 
+x = output.replace("Manufacturer","")
+print (x)
+if x == "VMware, Inc." or "": #virtualbox command response needed.
```

Comment:
I don't think this line does what you mean for it to do, to evaluate multiple possible conditions with an or statement you use syntax like this:

`if x == "value1" or x == "value2"`

Reply:
I'm also missing a bit here because I don't use Virtual Box so I don't know what the manufacturer's response is but thanks for that help.